platform/quality-assurance/auth-session-loadtest.md

```diff
@@ -21,10 +21,11 @@
     ![Jenkins console output example][jenkins-console-output]
 1. Scroll through the output until you see an array of hashes with keys "uuid" and "cookie_header"  
     ![Jenkins session cookie output example][jenkins-session-cookie-output]
-1. Trim unecessary characters such as escaping until you end up with something like the below  
-    ![Jenkins trimmed session cookies example][jenkins-trimmed-session-cookies]
-1. Save the trimmed array to a file named `sessions.json`
+1. Copy the entire array, including the lines with square-brackets  
+    ![Jenkins session cookie output selection example][jenkins-session-cookie-output-selected]
+1. Paste the copied array into a new file and save as `sessions.json`
 1. Use the array of sessions in `sessions.json` with Python's [json library](https://docs.python.org/3/library/json.html)
+    * For running a load-test locally ([see Devops Load-Testing README](https://github.com/department-of-veterans-affairs/devops/tree/master/loadtest/README.md)), save the file into the same folder as your load-test file.
```

Comment:
@peterhill-oddball , feel free to delete this insertion -- I just wasn't sure how the Python json lib relates to actually running devops repo’s load-test locally.

Reply:
I was just referencing the json lib for people who might be new to Python (like me) and need a quick reference for how to open a json file in Python and deal with the key-value pairs.  Your inserted information is good though!